src/Symfony/Component/Console/Command/LockableTrait.php

```diff
@@ -13,9 +13,10 @@
 
 use Symfony\Component\Console\Exception\LogicException;
 use Symfony\Component\Lock\Factory;
-use Symfony\Component\Lock\Lock;
+use Symfony\Component\Lock\LockInterface;
 use Symfony\Component\Lock\Store\FlockStore;
 use Symfony\Component\Lock\Store\SemaphoreStore;
+use Symfony\Component\Lock\StoreInterface;
```

Comment:
beware, the interface has been renamed in https://github.com/symfony/symfony/pull/32198